.github/workflows/release.yml

```diff
@@ -103,45 +103,60 @@ jobs:
         with:
           github-token: ${{ secrets.MOVE2KUBE_PATOKEN }}
           script: |
+            core.info('DEBUG 1');
```

Comment:
can we have some informative messages here? Applies to all the `core.info()` messages.

Reply:
done